riemann/streams/nuvlabox-streams.clj

```diff
@@ -0,0 +1,63 @@
+(require '[sixsq.slipstream.api.service-offer :as service-offer])
+
+;(logging/init {:file "/var/log/riemann/riemann.log"})
+;
+;(let [host "0.0.0.0"]
+;     (tcp-server {:host host})
+;     (udp-server {:host host})
+;     (ws-server  {:host host}))
+
+(def expiration-watch       1)
+(def default-ttl            5)
+(def nuvlabox-tag           "nuvlabox")
+(def nuvlabox-ping-service  "load/load/shortterm")
```

Comment:
My understanding of the implementation was that we use any received metrics since if we receive at least one metric it means that the connection works.
Maybe a future improvement ?